seeds/scripts/rerunners/sdvx/scrape-maya2s.ts

```diff
@@ -89,9 +168,15 @@ async function main() {
 	const songs: SDVXSong[] = ReadCollection("songs-sdvx.json");
 
 	for (const level of [17, 18, 19]) {
-		console.log(`\nScraping Level ${level}`);
+		console.log(`\n[S Rank] Scraping Level ${level}`);
+
+		await scrape(`https://sdvx.maya2silence.com/table/${level}`, charts, songs, level);
+	}
+
+	for (const level of [16, 17, 18, 19, 20]) {
+		console.log(`\n[PUC Lamp] Scraping Level ${level}`);
 
-		await scrape(charts, songs, level);
+		await scrape(`https://sdvx.maya2silence.com/table/${level}p`, charts, songs, level, false);
```

Comment:
Ditto on the boolean stuff here. It's unecessarily hard to look at this function and see what it's doing, and what the `false` argument is without looking it up. The last argument should be a union type `"S_TIER" | "PUC_TIER"` and not be default.